chia/cmds/rpc.py

```diff
@@ -0,0 +1,139 @@
+import asyncio
+import json
+import sys
+from typing import Any, Dict, List, Optional, TextIO
+
+import click
+from aiohttp import ClientResponseError
+
+from chia.rpc.rpc_client import RpcClient
+from chia.util.config import load_config
+from chia.util.default_root import DEFAULT_ROOT_PATH
+from chia.util.ints import uint16
+
+services: List[str] = ["farmer", "full_node", "harvester", "wallet"]
```

Comment:
I considered that both initially, but for `crawler` it doesn't work currently unless i add some ugly hack to pull the port because its inside the `seeder` config. For `timelord` i wasn't very convinced to add it because it doesn't provide any useful endpoint besides the standard ones to handle connections and stop it but i take a look at it later again, im afk now for few hours.

Reply:
@ChiaMineJP See f59bb3d281e731e86edaa5e75d58693e6005420f, i added both :)